### DISCUSSION TITLE
Error:This user account is blocked.

### DISCUSSION THREAD

**Discussion by Nikita7795** [vercel/next.js#18024] [Help] [answered]
Error:This user account is blocked. Contact support@vercel.com for more information.

why does he write this way?

**Comment by mcsdevv** [comment, marked as answer]
Hey @Nikita7795, please follow the instruction provided and we'll be glad to help out.

**Reply by YuriSizuku** [reply]
> Hey @Nikita7795, please follow the instruction provided and we'll be glad to help out.

I also have this problem, even though this is my first time to try to use vercel. 

Error:This user account is blocked. Contact support@vercel.com for more information.

**Reply by yy2lyx** [reply]
I also have this problem, even though this is my first time to try to use vercel.Please help me out.
Error:This user account is blocked. Contact support@vercel.com for more information.

**Reply by mcsdevv** [reply]
Hello both, please follow the instructions provided in the message and we will be glad to help you.

**Reply by SergeyKrasnolobov** [reply]
hi, actually there is no any instructions provided in the message. Even there is no any form to discuss this issue with support team!!!

**Reply by mcsdevv** [reply]
> hi, actually there is no any instructions provided in the message. Even there is no any form to discuss this issue with support team!!!

Are you able to get in touch at the address mentioned?

**Reply by SergeyKrasnolobov** [reply]
yep

**Reply by mcsdevv** [reply]
Can you clarify which instructions were missing?

**Comment by tootal** [comment]
Error:This user account is blocked. Contact support@vercel.com for more information.
I am the first time to try vercel.

**Reply by evan8623** [reply]
me too

**Reply by leerob** [reply]
Hey! Please follow the instructions provided in the message and we will be glad to help you.
